backend/clickhouse/query.go

```diff
@@ -516,6 +512,20 @@ func readMetrics[T ~string](ctx context.Context, client *Client, sampleableConfi
 		metricExpr = "1.0"
 	}
 
+	needsValue := false
+	for _, metricType := range metricTypes {
```

Comment:
does the `tableConfig.defaultFilter` exclude the `IsIngestedBy` traces for sampling?

Reply:
yep, this is a little hacky but this skips applying the default `<> Internal` filter